src/pages/collaborations/collaborationManagementForm.jsx

```diff
@@ -3,15 +3,50 @@ import { FormattedMessage, useIntl } from 'react-intl';
 import { get, some } from 'lodash-es';
 import { TextField } from '@material-ui/core';
 import { Autocomplete } from '@material-ui/lab';
+import { useQueryClient } from 'react-query';
+
+import queryKeys from '../../constants/queryKeys';
 import Button from '../../components/Button';
 import CustomAlert from '../../components/Alert';
 import useEstablishCollaborationAsUserManager from '../../models/collaboration/useEstablishCollaborationAsUserManager';
 
+function mutuallyRevoked(members) {
```

Comment:
i suggest moving some of these functions to a utility file, this component is a bit of a monster

Reply:
Done.